dags/legacy_test/tests/pytorch/r2.5.1/common.libsonnet

```diff
@@ -67,7 +67,7 @@ local volumes = import 'templates/volumes.libsonnet';
 
                 ctc = cloud_tpu_client.Client(tpu=os.path.basename('$(TPU_NAME)'), zone=os.path.dirname('$(TPU_NAME)'))
                 ctc.wait_for_healthy()
-                ctc.configure_tpu_version(f'pytorch-2.5-dev{libtpu_date}', restart_type='always')
+                ctc.configure_tpu_version(f'pytorch-2.5.1-dev{libtpu_date}', restart_type='always')
```

Comment:
what `libtpu_date` is used for the record?

Reply:
`libtpu_date` is generated by pattern match([link](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/7e941d31de1f5665839263488a54db46b8d46afd/dags/legacy_test/tests/pytorch/r2.5.1/common.libsonnet#L57C1-L65C48)) from `torch_xla` deps list:
```
(torch310) manfei@xxx:~/pytorch/xla$ PJRT_DEVICE=TPU python
Python 3.10.15 (main, Oct  3 2024, 07:27:34) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> import torch_xla
>>> import importlib_metadata
>>> print(importlib_metadata.requires('torch_xla'))
['absl-py>=1.0.0', 'numpy', 'pyyaml', 'requests', 'importlib_metadata>=4.6; python_version < "3.10"', 'libtpu-nightly==0.1.dev20241020; extra == "tpu"', 'tpu-info; extra == "tpu"', 'jaxlib==0.4.35.dev20241020; extra == "pallas"', 'jax==0.4.35.dev20241020; extra == "pallas"']
>>> exit()
(torch310) manfei@xxx:~/pytorch/xla$ 
```

so here `libtpu_date` should be 10/20

thanks for @zpcore's colloboartion